modules/apps/object/object-api/src/main/java/com/liferay/object/system/BaseSystemObjectDefinitionManager.java

```diff
@@ -100,20 +100,22 @@ protected Map<Locale, String> createLabelMap(String labelKey) {
 	}
 
 	protected ObjectField createObjectField(
-		String businessType, String dbType, String labelKey, String name,
-		boolean required, boolean system) {
+		String businessType, String dbType, String externalReferenceCode,
+		String labelKey, String name, boolean required, boolean system) {
```

Comment:
@seltong please reorder so that `externalReferenceCode` comes before the `businessType`